cpp/README.md

```diff
@@ -33,6 +33,7 @@ The following table lists the dependencies that are used. Most of them are requi
 | JsonCpp | 1.9.6    | No       | Yes (git repo)        | https://github.com/open-source-parsers/jsoncpp |
 | HDF5    | 1.12.0   | No       | No                    | https://www.hdfgroup.org/, package libhdf5-dev on apt (Ubuntu) |
 | GoogleTest | 1.10  | For Tests only | Yes (git repo)  | https://github.com/google/googletest |
+| LibSBML | 5.18.0 | For SBML integration only | Yes (sourgeforce repo) |  https://sbml.org/software/libsbml/ |

```

Comment:
i would prefer to fill Required with `No`. And also apply this to the GoogleTest part. Would make it more consistent with the libaries above. The current text could be put into the notes